src/controllers/UserController.js

```diff
@@ -29,4 +32,48 @@ export default class UserController {
       user: updatedUser
     });
   }
+
+  /**
+   * Fetch all users and roles
+   *
+   * @param {Object} req express request object
+   * @param {Object} res express response object
+   * @returns {*} success response
+   * @throws {*} error if database error
+   */
+  static async getAllRoles(req, res) {
+    const roles = await db.User.findAll({
```

Comment:
There should not be queries in the controller

Reply:
Changes applied